tango/integrations/transformers/model.py

```diff
@@ -31,3 +47,13 @@ def from_config(cls, config: Config, **kwargs) -> Model:
         Model.register("transformers::" + name + "::from_config", constructor="from_config")(
             wrapped_cls
         )
+
+for name, cls in modeling_flax_auto.__dict__.items():
+    if isinstance(cls, type) and name.startswith("FlaxAutoModel"):
```

Comment:
I'm still not sure how to express the flax - transformer - torch dependency situation. Let's bring this up with the team internally.